go-runtime/compile/build-template/.ftl.tmpl/go/main/main.go.tmpl

```diff
@@ -34,22 +34,7 @@ func init() {
 		reflection.ProvideResourcesForVerb(
             {{.TypeName}},
             {{- range .Resources}}
-                {{- with getVerbClient . }}
```

Comment:
would prefer to leave these in the template for readability, but with the new `MappedHandle` type, we need to recurse back into this logic to get the string for the resource that will be mapped - so pulled it out into a Go function